dspace-api/src/main/java/org/dspace/statistics/util/LocationUtils.java

```diff
@@ -62,7 +62,7 @@ static public String getContinentCode(String countryCode)
         String continent = countryToContinent.getProperty(countryCode);
         if (null == continent)
         {
-            logger.error("Unknown country code " + countryCode);
+            logger.info("Unknown country code " + countryCode);
```

Comment:
I've noticed "EU" coming through as a country quite a lot. While that's technically not correct, perhaps we should just add an EU->EU mapping to country-continent-codes.properties? Though I guess this would have follow-on effects where the country code is translated back to a display name.